### PR TITLE
[FIX] point_of_sale: apply quantity bracode nomenclature

### PR DESCRIPTION
**Pull request by HANNICHE-Walid**
Steps to reproduce:
1) Have Barcode Installed and have Barcode Nomenclature set 
2) edit 1st rule(Price Barcodes 2 Decimals) to type Quantity
3) Create a product with barcode 2380201000008
4) Open POS and scan barcode 2380201010007

Bug:
error popup product not found because the callback for this rule is not set

Fix:
apply same callback as weight (set product quantity)

opw-3213595

### CODE REVIEW

**Comment by robinengels** [comment]
LGTM

**Comment by HANNICHE-Walid** [comment]
hi @caburj can you check this PR please

File: addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js [hunk 1]
```diff
@@ -221,7 +222,7 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
                         price_manually_set: true,
                     },
                 });
-            } else if (code.type === 'weight') {
+            } else if (code.type === 'weight' | code.type === 'quantity') {
```
Comment (caburj) [inline addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js:225]:
```suggestion
            } else if (code.type === 'weight' || code.type === 'quantity') {
```

**Review by caburj — CHANGES_REQUESTED** [review]
Looking good.

File: addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js (hunk 1 above)
Comment (HANNICHE-Walid) [reply addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js]: done :)

**Comment by caburj** [comment]
You might need to rebase to get rid of the documentation CI error.

**Pull request closed without merging**